CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift

```diff
@@ -0,0 +1,66 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import InstructorViews
+import SwiftUI
+
+/// View displaying information about Abhinav Agarwal
+public struct AbhinavAgarwal: View {
+    @Binding var navigationPath: NavigationPath
+    private let timer = Timer.publish(every: 1.5, on: .main, in: .common).autoconnect()
+    @State var animationTick = true
+    public var body: some View {
```

Comment:
Please add two empty lines between stored properties and computed properties in your types:
```suggestion
    @State var animationTick = true
    
    
    public var body: some View {
```